### PR TITLE
Add an initial Publisher FAQ

### PR DESCRIPTION
**Pull request by ericholscher**
This is a start to answering all the common questions I get.

### CODE REVIEW

File: content/pages/advertising-faq.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 Title: Frequently Asked Questions
-slug: faq
```
Comment (ericholscher) [inline content/pages/advertising-faq.md:2]: Seems this is only needed for translation? 

https://docs.getpelican.com/en/3.6.3/content.html#translations

> Pelican uses the article’s URL “slug” to determine if two or more articles are translations of one another.

File: ethicalads-theme/templates/includes/footer.html [hunk 2]
```diff
@@ -88,6 +88,9 @@ <h6 class="font-weight-bold text-uppercase text-gray-700">Advertisers</h6>
           <li class="mb-3">
             <a href="/advertisers/faq/" class="text-reset">Advertiser FAQ</a>
           </li>
+          <li class="mb-3">
+            <a href="https://www.ethicalads.io/prospectus/ethicalads-advertiser-prospectus.pdf" class="text-reset">Advertiser Prospectus</a>
```
Comment (ericholscher) [inline ethicalads-theme/templates/includes/footer.html:92]: The Publisher footer seemed heavy vs. the ads, so I added this.

File: content/pages/publisher-faq.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+The later you are in the day,
+the more likely that you might see more unpaid ads if we haven't fully sold your specific location and keywords.
+You can always look in your publisher dashboard to see more information on what is happening, once data has started to be collected.
+
+
+## How do you target ads?
+
+We target ads on two primary things: country and keywords.
+Country targeting uses the IP address of the client,
+but we don't allow any more granular targeting than the country.
```
Comment (davidfischer) [inline content/pages/publisher-faq.md:61]: We mention state level targeting for the US in the advertiser FAQ. We should remove one of them.

File: content/pages/advertising-faq.md (hunk 1 above)
Comment (davidfischer) [reply content/pages/advertising-faq.md:2]: It is used to construct the URL but I guess it isn't necessary with `save_as` and `url`.

**Review by davidfischer — APPROVED** [review]

File: content/pages/publisher-faq.md (hunk 3 above)
Comment (ericholscher) [reply content/pages/publisher-faq.md]: I'll leave it for now. We can adapt if we get pushback on this, but I don't see it.

**Pull request merged**